spec/std/socket/tcp_server_spec.cr

```diff
@@ -33,7 +33,7 @@ describe TCPServer do
       end
 
       it "raises when port is negative" do
-        expect_raises(Socket::Error, linux? ? "getaddrinfo: Servname not supported for ai_socktype" : "No address found for #{address}:-12 over TCP") do
+        expect_raises(Socket::Error, linux? ? (musl? ? "getaddrinfo: Unrecognized service" : "getaddrinfo: Servname not supported for ai_socktype") : "No address found for #{address}:-12 over TCP") do
```

Comment:
This line is getting a bit long, how about checking for errno by number instead of by message?

Reply:
I'm not sure... Error numbers have portability issues.

Reply:
We have constants under `Errno` for the values. We should do the same for `Socket::Error`.